controllers/opentelemetrycollector_controller.go

```diff
@@ -79,13 +79,16 @@ type Params struct {
 func (r *OpenTelemetryCollectorReconciler) Validate(otelcol v1beta1.OpenTelemetryCollector) admission.Warnings {
 	var warnings admission.Warnings
 
-	params, err := r.getParams(otelcol)
-	if err != nil {
-		warnings = append(warnings, err.Error())
-		return warnings
+	cfg := config.New(
+		config.WithCollectorImage("default-collector"),
+		config.WithTargetAllocatorImage("default-ta-allocator"),
+	)
+	params := manifests.Params{
+		Log:     logr.Discard(),
+		Config:  cfg,
```

Comment:
i think for the config we can propagate that https://github.com/open-telemetry/opentelemetry-operator/blob/b6f0911cda24680d2f1ae521d469b01b4f4e13e6/controllers/opentelemetrycollector_controller.go#L194